src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -296,6 +297,11 @@ public CompletableFuture<Void> close(boolean drop) {
                   this.owningClient.verifyChannelsAreFullyCommitted(
                       Collections.singletonList(this));
 
+              // Deregister per-table metrics.
+              String metricBaseName = MetricRegistry.name("table_blob", "size", "histogram");
+              String metricName = String.format("%s,db=%s,schema=%s,table=%s", metricBaseName, getDBName(), getSchemaName(), getTableName());
```

Comment:
I wonder if we could factor that out to avoid duplication in FlushService.java

Reply:
Sure, we can do that.